src/sass/components/_product-box.scss

```diff
@@ -11,6 +11,7 @@
       size: contain;
       position: center;
       repeat: no-repeat;
+      /*image: url("/images/beds/bed01.png");*/
```

Comment:
To jeszcze tylko usuń tę jedną linijkę i możemy merge'ować ;)

Reply:
Linijka została usunięta :)